chemclipse/plugins/org.eclipse.chemclipse.msd.model/src/org/eclipse/chemclipse/msd/model/matrix/ExtractedMatrix.java

```diff
@@ -87,62 +87,63 @@ private Boolean checkHighRes( int limit ) {
 				.collect(Collectors.toList()); //
 		return (scans);
 	}
-	
+
 	private int[] getMinMaxMz() {
-		
-		int[] minMaxMz = new int[2];
 
+		int[] minMaxMz = new int[2];
 		Stream<Double> s = scans.stream() //
 				.flatMap(scan -> scan.getIons().stream()) //
-				.map(x -> (Double) x.getIon());
+				.map(x -> (Double)x.getIon());
 		DoubleSummaryStatistics d = s.collect(Collectors.summarizingDouble(value -> value));
-		
 		minMaxMz[0] = AbstractIon.getIon(d.getMin());
 		minMaxMz[1] = AbstractIon.getIon(d.getMax());
-		return (minMaxMz); 
+		return (minMaxMz);
 	}
 
 	public float[][] getMatrix() {
+
 		return this.signal;
 	}
 
 	public int getNumberOfScans() {
+
 		return this.numberOfScans;
 	}
 
 	public int getNumberOfIons() {
+
 		return this.numberOfIons;
 	}
 
-	public void updateSignal(float[][] signal, int numberOfScans, int nubmerOfIons ) {
-		
+	public void updateSignal(float[][] signal) {
```

Comment:
If you want going further, you can even omit the signal matrix here as it is already available in this.signal. At least you should not rely on the fact that the passed in array has the desired size (never trust the caller!)